core-services/prow/02_config/_plugins.yaml

```diff
@@ -837,6 +837,11 @@ config_updater:
         app.ci:
         - ci
       name: ci-operator-misc-configs
+    ci-operator/config/**/windows-machine-config-bootstrapper/*community-4.6*.yaml:
```

Comment:
Shouldnt it follow the same regex as `ci-operator/config/**/windows-machine-config-operator/*-community-*.yaml`, so that future community branches are handled as well?

Reply:
With operator replaced with bootstrapper of course

Reply:
The regex matcher returns multiple files if I do that 
```
make prow-config
docker pull registry.svc.ci.openshift.org/ci/determinize-prow-config:latest
latest: Pulling from ci/determinize-prow-config
Digest: sha256:3ad5b874128f5f8f90b51c53a99c14b972cd6f2b7213d38047d39958dd1e1d2b
Status: Image is up to date for registry.svc.ci.openshift.org/ci/determinize-prow-config:latest
registry.svc.ci.openshift.org/ci/determinize-prow-config:latest
docker run --rm -v "/home/vaishnavi/dev/openshift/installer/release/core-services/prow/02_config:/config:z" registry.svc.ci.openshift.org/ci/determinize-prow-config:latest --prow-config-dir /config
time="2021-01-06T15:20:34Z" level=fatal msg="could not update Prow plugin configuration" error="could not load Prow plugin configuration: key *-community-*.yaml in configmap ci-operator-misc-configs updated with more than one file"
make: *** [Makefile:80: prow-config] Error 1

```

Reply:
I could add `*community*` instead  to handle future community branches